Mathlib/Algebra/Lie/Normalizer.lean

```diff
@@ -72,11 +72,11 @@ theorem normalizer_inf : (N₁ ⊓ N₂).normalizer = N₁.normalizer ⊓ N₂.n
 #align lie_submodule.normalizer_inf LieSubmodule.normalizer_inf
 
 @[mono]
-theorem monotone_normalizer : Monotone (normalizer : LieSubmodule R L M → LieSubmodule R L M) := by
-  intro N₁ N₂ h m hm
+theorem monotone_normalizer (h : N₁ ≤ N₂) : normalizer N₁ ≤ normalizer N₂ := by
```

Comment:
Instead of replace `monotone_normalizer`, can you add that as a new lemma called `normalizer_mono` (and tag that with `mono`)?